src/fluent-plugin-mdsd/Makefile

```diff
@@ -14,9 +14,11 @@ all: $(MDSDGEM)
 
 $(MDSDGEM): $(MDSDGEM).gemspec lib/fluent/plugin/out_mdsd.rb
 	cp -f ../builddir/release/lib/Liboutmdsdrb.so lib/fluent/plugin/Liboutmdsdrb.so
+	cp -f ../../LICENSE.txt .
+	cp -f ../../README.md .
 	$(GEM) build $(MDSDGEM).gemspec
```

Comment:
removed this Makefile. moved the code to build script

---
In reply to: [114166784](https://github.com/Azure/fluentd-plugin-mdsd/pull/26#discussion_r114166784) [](ancestors = 114166784)